### DISCUSSION TITLE
历史界面切换问题

### DISCUSSION THREAD

**Discussion by MianJu28** [the1812/Bilibili-Evolved#2729] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/68192943/146509191-dbf68ed6-8e66-44bb-9ef9-ae4952e7a75c.png)

历史界面之前是按类型分类，直接点就可以看同一类的历史记录，

![image](https://user-images.githubusercontent.com/68192943/146508012-23758b55-e246-4a71-8501-612ef9557705.png)
现在改成按类型过滤了，要看某一类历史记录要把其他项关掉，有点不方便，能不能把之前的加上

**Comment by asgh215** [comment]
确实，我也感觉这个不太方便，找了一圈发现是自定义顶栏这个样式的问题，目前我不启用这个样式，希望后续能在这个样式更新分页查看历史的选项

**Reply by asgh215** [reply]
另外在关闭自定义顶栏这个样式时也不能启用隐藏顶部横幅这一样式，在我这里出现了顶栏文字消失的问题
